src/common/translations/state.ts

```diff
@@ -0,0 +1,30 @@
+import type { HassConfig, HassEntity } from "home-assistant-js-websocket";
+import type { EntityRegistryDisplayEntry } from "../../data/entity_registry";
+import type { LocalizeFunc } from "./localize";
+import type { FrontendLocaleData } from "../../data/translation";
+
+export type FormatStateFunc = (
+  stateObj: HassEntity,
+  entity?: EntityRegistryDisplayEntry,
+  state?: string
+) => string;
+
+export const computeFormatState = async (
+  localize: LocalizeFunc,
+  locale: FrontendLocaleData,
+  config: HassConfig
+): Promise<FormatStateFunc> => {
+  const { computeStateDisplaySingleEntity } = await import(
+    "../entity/compute_state_display"
+  );
```

Comment:
Why we do we do this async?

Reply:
`compute_state_display` import many things so I preferred to lazy load it. Do you think we can import it directly?